deploy/example/sharedisk/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: managed-csi-sharedisk
```

Comment:
Use name `azuredisk-csi-premium-lrs-shared` to clearly tie it to Azure PremiumLRS disks.

Reply:
Done